app/routes/task_routes.py

```diff
@@ -0,0 +1,140 @@
+from flask import Blueprint, request, jsonify, make_response, abort 
+from app import db
+from app.models.task import Task
+from datetime import datetime 
+from sqlalchemy.types import DateTime
+from sqlalchemy.sql.functions import now
+import requests, os
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
```

Comment:
Here's a good opportunity for some refactoring! Since you're defining the same function in two route files, you could create a new file `route_helpers.py` where that function is defined and then import it wherever you need!

Reply:
I wanted to do this with a little more time! I think I'll go back and add it!